freshlybuiltimagebol/downloader.py

```diff
@@ -2,7 +2,7 @@
 from os import mkdir,path,remove
 from tqdm import tqdm
 from hashlib import md5
-from colorama import Fore
+import colorama as col
 

```

Comment:
Use from statement rather than import

Reply:
here we require to import the entire class otherwise it will cause a bug